app/User.php

```diff
@@ -129,4 +135,32 @@ public function updateCollaboratorNames()
                 $collaborator->update(['name' => $this->name]);
             });
     }
+
+    public function updateCollaboratorGithubUsernames()
+    {
+        $this->collaborators
+            ->reject(function ($collaborator) {
```

Comment:
This isn't really necessary, because the next filter would catch any null github_user_ids, right?

Reply:
This was to avoid updating Github usernames in cases where both the collaborator `github_user_id` and the user `github_user_id` are null as tested in `collaborator_github_usernames_are_only_updated_when_github_user_id_is_set`.  

I just reworked this a bit to replace the call to `reject` on the collaborators collection with a `! $this->github_user_id` guard clause.  I think this will make it more clear what the intent is as well as match what is described in the test name.